projects/Generic/filesystem/usr/lib/systemd/system/rtk_bt.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Realtek RTL8723BS bluetooth service
+ConditionPathExists=/dev/ttyS1
```

Comment:
I believe I can replace the condition check with:
ConditionPathExists=/sys/devices/platform/80860F0A:00/OBDA8723:00